src/test/java/CandidatesTest.java

```diff
@@ -0,0 +1,29 @@
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class CandidatesTest {
+    @Test
+    void create(){
+        assertThat(new Candidates().size()).isEqualTo(0);
+    }
+
+    @Test
+    void calculate_benefit_when_buy_three_lottos(){
+        Candidates candidates = new Candidates();
+
+        candidates.add(new Lotto(new ArrayList<>(Arrays.asList(1, 2, 3, 4, 13, 14)))); // match count : 4
```

Comment:
```suggestion
        candidates.add(new Lotto(1, 2, 3, 4, 13, 14));
```

Lotto의 생성자로 가변 인자를 제공해 위와 같이 Lotto를 쉽게 생성하도록 지원하는 것은 어떨까?